fanuc_cr7ial_moveit_config/config/fanuc_cr7ial.srdf

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" ?>
+<!--This does not replace URDF, and is not an extension of URDF.
+    This is a format for representing semantic information about the robot structure.
+    A URDF file must exist for this robot as well, where the joints and the links that are referenced are defined
+-->
+<robot name="fanuc_cr7ial">
+    <!--GROUPS: Representation of a set of joints and links. This can be useful for specifying DOF to plan for, defining arms, end effectors, etc-->
+    <!--LINKS: When a link is specified, the parent joint of that link (if it exists) is automatically included-->
+    <!--JOINTS: When a joint is specified, the child link of that joint (which will always exist) is automatically included-->
+    <!--CHAINS: When a chain is specified, all the links along the chain (including endpoints) are included in the group. Additionally, all the joints that are parents to included links are also included. This means that joints along the chain and the parent joint of the base link are included in the group-->
+    <!--SUBGROUPS: Groups can also be formed by referencing to already defined group names-->
+    <group name="manipulator">
+        <chain base_link="base_link" tip_link="tool0" />
+    </group>
+    <!--GROUP STATES: Purpose: Define a named state for a particular group, in terms of joint values. This is useful to define states like 'folded arms'-->
+    <group_state name="all-zeroes" group="manipulator">
+        <joint name="joint_1" value="0" />
+        <joint name="joint_2" value="0" />
+        <joint name="joint_3" value="0" />
+        <joint name="joint_4" value="0" />
+        <joint name="joint_5" value="0" />
+        <joint name="joint_6" value="0" />
+    </group_state>
+    <group_state name="ship" group="manipulator">
+        <joint name="joint_1" value="0" />
+        <joint name="joint_2" value="-0.6108" />
+        <joint name="joint_3" value="-0.6108" />
+        <joint name="joint_4" value="0" />
+        <joint name="joint_5" value="-1.4311" />
+        <joint name="joint_6" value="0" />
+    </group_state>
+    <!--VIRTUAL JOINT: Purpose: this element defines a virtual joint between a robot link and an external frame of reference (considered fixed with respect to the robot)-->
+    <virtual_joint name="MountPoint" type="fixed" parent_frame="arm_mount_point" child_link="base_link" />
```

Comment:
It's really just a convention, and I don't expect people to re-use MoveIt configs as-is anyway, apart from some initial testing maybe (who has a single robot in complete isolation to plan for).

The "world" frame that you are referring to is what Fanuc calls the *World Frame*, which is modelled with the `base` frame in the urdf.